lib/types.ts

```diff
@@ -31,6 +31,7 @@ export type NoteEntity = Entity<Note>;
 export type Tag = {
   index?: number;
   name: TagName;
+  deleted?: boolean;
```

Comment:
while I don't fully know how to explain it I do have some reservations about making this appear as part of the tag data model. I feel like this is inherently a display property and something we're calculating on the fly purely by comparing a tag against the current known app state.

is it possible that we can get what we need in this PR without adding any new properties to `Tag`? with your new selector I don't see why anything down the line need to be aware of this, other than the tag list which needs to know to render them differently, though we can already compute that in the UI component itself.

Reply:
Yeah no problem, I've removed it in this [commit](https://github.com/Automattic/simplenote-electron/pull/2817/commits/292931b3a7c48dd4c49890048a0e4114d6834f82). Now the deleted state is calculated in the `NoteRevisions` component.